src/aes_hash.rs

```diff
@@ -160,6 +210,9 @@ impl Hasher for AHasher {
             self.hash_in(value.convert());
         } else {
             if data.len() > 32 {
+                if data.len() > 128 {
```

Comment:
This is sub-optimal. It's handling the >128 case but then the >64 case below is only for 65-128, which could be specialized be more efficient. 
I would be tempted to avoid the additional if check if possible as branch mispredictions are more common and costly than show up in benchmarks where the data is predictable. But I don't have a good sense on how much is lost operating on 4 blocks at a time as opposed  to 8.